dotcom-rendering/src/web/components/SignInGate/gateDesigns/SignInGateMainCheckoutComplete.tsx

```diff
@@ -0,0 +1,263 @@
+import { css } from '@emotion/react';
+import {
+	body,
+	brand,
+	from,
+	headline,
+	neutral,
+	space,
+} from '@guardian/source-foundations';
+import { Button, Link, LinkButton } from '@guardian/source-react-components';
+import { trackLink } from '../componentEventTracking';
+import type {
+	Product,
+	SignInGatePropsWithCheckoutCompleteCookieData,
+	UserType,
+} from '../types';
+import {
+	firstParagraphOverlay,
+	hideElementsCss,
+	registerButton,
+	signInGateContainer,
+} from './shared';
+
+const personalisedHeadingStyles = css`
+	${headline.small({ fontWeight: 'bold' })};
+	border-top: 2px ${brand[400]} solid;
+	${from.phablet} {
+		padding-right: 160px;
+		${headline.medium({ fontWeight: 'bold' })};
+	}
+	padding-bottom: ${space[2]}px;
+`;
+
+const personalisedBodyBold = css`
+	${body.medium({ fontWeight: 'bold' })}
+	${from.phablet} {
+		padding-right: 130px;
+	}
+	color: ${brand[400]};
+`;
+
+const bulletStyles = css`
+	text-indent: -30px; /* second line indentation */
+	margin-left: 30px; /* second line indentation */
+	color: ${neutral[100]};
+	display: flex;
+	flex-direction: column;
+	li:not(:first-of-type) {
+		margin-top: ${space[1]}px;
+	}
+	li::before {
+		content: '';
+		display: inline-block;
+		width: 12px;
+		height: 12px;
+		margin-right: ${space[4]}px;
+		background: ${brand[400]};
+		border-radius: 50%;
+	}
+`;
+
+const personalisedBodyTextList = css`
+	${body.medium({ fontWeight: 'medium' })};
+	color: black;
+`;
+
+const personalisedActionButtons = css`
+	display: flex;
+	flex-direction: row;
+	justify-content: flex-start;
+	margin-top: 20px;
+	margin-bottom: 20px;
+
+	> a {
+		/* stylelint-disable-next-line declaration-no-important */
+		margin-right: ${space[4]}px !important;
```

Comment:
Is there a reason for these `!importants`? We try to avoid them if at all possible, as they can make our code much harder to reason about and maintain

There is a cssOverrides property on the LinkButton component may allow you to do what is required, otherwise I'd consider looking to see if we can either achieve the styling some other way, or make an alternation to the design.

Reply:
Yeah, this is something that has come as a result of forking `SignInGateMain`. I know it's not the _best_ practise, but I feel that since we are already supporting it in this way in the code base, I would prefer to leave it as this way for now.

Reply:
Happy to make a note for us to revisit next time we're looking at the styling for the sign in gate main

Reply:
Ah I see, definitely would be a recommendation of something to look at if your team were to have a health week or something, but given that it's already used in the sign in gate component (I hadn't realised this, sorry!), I agree it's probably not worth the effort to change right now.

Reply:
yep, that's a good shout

Reply:
Made a ticket for this here: https://trello.com/c/io1zD3C2/389-remove-important-in-the-css-in-the-signingate-component